src/App.css

```diff
@@ -1,38 +1,83 @@
+@import url('https://fonts.googleapis.com/css?family=Comfortaa');
+
+:root
+{
+    --dark-grey     : #2F4858;
+    --dark-grey-2   : #282c34;
+    --royal-blue    : #29517C;
+    --sky-blue      : #0e7490;
+    --bluish-white  : #f6f6f6;
+    --pure-white    : white;
+    --red-color     : #ca0e1d;
+    --background-color-dull-yellow: #e0e0d4;
+    --yellow-background-color     : #efe6e1;
+    --light-grey    : #e4e6eb;
+    --dull-grey     : #8f8f8f;
+}
+
+html
+{
+    font-family: 'Comfortaa', cursive;
+}
+
+body
+{
+    font-family: 'Comfortaa', cursive;
+}
+
+html, body
+{
+    height: auto;
+    background-color: white;
+}
+
 .App {
-  text-align: center;
+    min-height: 100%;
+    text-align: center;
+    background-color: white;
+    font-family: 'Comfortaa', cursive;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+*::-webkit-scrollbar 
+{
+    height: 1em;
+}
+  
+*::-webkit-scrollbar-track 
+{
+    background: var(--light-grey);
+}
+  
+*::-webkit-scrollbar-thumb 
+{
+    background: var(--sky-blue);
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+a::after
+{
+    content: '';
+    display: block;
+    width: 0px;
+    height: 0px;
+    background: var(--dark-blue-color);
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+a:hover::after
+{
+    width: 0;
 }
 
-.App-link {
-  color: #61dafb;
+.page-container
+{
+    min-height: calc(100vh - 107.23px);
+    display: grid;
+    grid-template-columns: 25rem 1fr;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+@media only screen and (max-width:1015px){
```

Comment:
suggestion: use should use em for media queries

Reply:
Done